controllers/ajax_controller.py

```diff
@@ -3,16 +3,55 @@
 import json
 import time
 
+from base_controller import CacheableHandler, LoggedInHandler
 from google.appengine.api import memcache
+from google.appengine.ext import ndb
 from google.appengine.ext.webapp import template
-
-from base_controller import CacheableHandler
-
+from helpers.model_to_dict import ModelToDict
+from helpers.mytba_helper import MyTBAHelper
+from models.account import Account
 from models.event import Event
+from models.favorite import Favorite
 from models.sitevar import Sitevar
 from models.typeahead_entry import TypeaheadEntry
 
 
+class AccountFavoritesHandler(LoggedInHandler):
+    """
+    For getting, adding, and deleting an account's favorites
+    """
+    def get(self):
+        if not self.user_bundle.user:
+            self.response.set_status(401)
+            return
+
+        favorites = Favorite.query(ancestor=ndb.Key(Account, self.user_bundle.user.user_id())).fetch()
+        self.response.out.write(json.dumps([ModelToDict.favoriteConverter(fav) for fav in favorites]))
+
+    def post(self):
+        if not self.user_bundle.user:
+            self.response.set_status(401)
+            return
+
+        action = self.request.get("action")  # 'add' or 'delete'
```

Comment:
Why did you choose this over separate `AccountFavoritesAddHandler` and `AccountFavoritesDeleteHandler`?